src/options.py

```diff
@@ -52,6 +52,9 @@ def initialize(self):
                             action='store_true', default=False)
         self.parser.add_argument('--readme', '-r', help=('displays readme file content for '
                             'more help details'), required=False, action='store_true')
+        self.parser.add_argument('--cbz', required=False,
```

Comment:
Since `--separate` is required for `--cbz` it might be worth including something like the below, when the arguments are read in the parse function
```
class DependentArgumentsError(Exception):
    '''Exception raised when an argument is dependent on another arguments'
    
    def __init__(self, arg1, arg2):
        self.arg1 = arg1
        self.arg2 = arg2
        self.message = f"Argument {arg1} is dependent on {arg2}"
        super().__init__(self.message)
    pass


if self.args.cbz and self.args.separate == None:
  self.parser.print_help()
  raise DependentArgumentsError(['--cbz'], ['--separate'])
```

Reply:
I made the change you suggested. Good idea.